docs/content/getting-started/code-generation.md

```diff
@@ -239,27 +237,27 @@ There are two ways to define and use partial templates:
 - Use the `{{ define "name" }}` syntax in a regular template
 - User the `-P` to load them from a file
 
-Let's extract field generation into its own template, where we _could_ make it complex.
+Let's extract field generation into its template, where we _could_ make it complex.
 We won't here, but an example is struct tags for our Go fields.
-We can also use template helpers in the output filepath.
+We can also use template helpers in the output file path.
 
 {{<codeInner title="> terminal" lang="sh">}}
 $ hof gen data.cue schema.cue -P field.go -T types.go -O out/
 {{</codeInner>}}
 
 {{<codePane3
-	title1="types.go" file1="code/getting-started/code-generation/typeP.go" lang1="go"
-	title2="field.go" file2="code/getting-started/code-generation/field.go" lang2="go"
-	title3="out/types.go" file3="code/getting-started/code-generation/out/types.go" lang3="text"
+  title1="types.go" file1="code/getting-started/code-generation/typeP.go" lang1="go"
```

Comment:
same questions about = space here, but it is inconsistent

generally, I'd like to not have the space, it is weird that is only on one side of the = too